lib/shared/src/sourcegraph-api/completions/types.ts

```diff
@@ -20,6 +20,19 @@ export interface Message {
     // Note: The unified API only supports one system message passed as the first message
     speaker: 'human' | 'assistant' | 'system'
     text?: PromptString
+    cache_control?: boolean
+}
+
+export interface CompletionUsage {
+    completion_tokens: number
+    prompt_tokens: number
+    total_tokens: number
+    prompt_tokens_details?: PromptTokensDetails
+}
+
+export interface PromptTokensDetails {
+    cached_tokens: number
```

Comment:
I don't think this is 100% necessary but it doesn't hurt to handle undefined. We should probably use `?: number | null` everywhere for evolving fields to be properly robust in the face of unexpected values
```suggestion
    cached_tokens?: number
```

Reply:
done!